charts/scalardb-cluster/templates/scalardb-cluster/deployment.yaml

```diff
@@ -97,20 +93,9 @@ spec:
             - name: scalardb-cluster-database-properties-volume
               mountPath: /scalardb-cluster/node/scalardb-cluster-node.properties
               subPath: scalardb-cluster-node.properties
-            {{- if .Values.scalardbCluster.tls.caRootCertSecret }}
-            - name: scalardb-cluster-tls-ca-root-volume
-              mountPath: /tls/certs/ca-root-cert.pem
-              subPath: ca-root-cert
-            {{- end }}
-            {{- if .Values.scalardbCluster.tls.certChainSecret }}
-            - name: scalardb-cluster-tls-cert-chain-volume
-              mountPath: /tls/certs/cert-chain.pem
-              subPath: cert-chain
-            {{- end }}
-            {{- if .Values.scalardbCluster.tls.privateKeySecret }}
-            - name: scalardb-cluster-tls-private-key-volume
-              mountPath: /tls/certs/private-key.pem
-              subPath: private-key
+            {{- if .Values.scalardbCluster.tls.enabled }}
+            - name: scalardb-cluster-tls-volume
+              mountPath: /tls/scalardb-cluster/certs
```

Comment:
I combine all private keys and certificates into one volume to remove the `subPath` configuration.